### ISSUE TITLE
ReactUI is missing focusManager

### ISSUE THREAD

**Issue by jbalsas** [liferay/alloy-editor#113]
Currently, the React UI can't handle keyboard navigation between toolbars and buttons properly as it's missing a focus manager.
